### DISCUSSION TITLE
Feature request: saving a bundle document with the original indirect filename

### DISCUSSION THREAD

**Discussion by aamodio** [RussCoder/djvujs#56] [Ideas]
Hi RussCode,

When saving an indirect file in bundle format the original indirect filename should be preserved. 

Thanks,
A.

**Comment by RussCoder** [comment]
Yes, it can be done. I didn't preserve the original file name, because in most cases it's not provided programmatically (e.g. in the case of the browser extension) it is just "index.djvu". But if it is, it's worth preserving it.

**Comment by RussCoder** [comment]
Now the original file name is preserved in most cases (if it's not `index.djvu`). You need the viewer v.0.5.6

**Reply by aamodio** [reply]
Thanks, just installed!
